src/BusinessLogic/Http/Proxy.php

```diff
@@ -200,11 +201,48 @@ public function searchLocations($platformCountry, $postalZone, $query)
      */
     public function getPostalCodes($countryCode, $zipCode)
     {
-        $response = $this->call(HttpClient::HTTP_METHOD_GET, "locations/postalcodes/$countryCode/$zipCode");
+        $response = $this->call(HttpClient::HTTP_METHOD_GET, urlencode("locations/postalcodes/$countryCode/$zipCode"));
 
         return PostalCode::fromArrayBatch($response->decodeBodyAsJson());
     }
 
+    /**
+     * Returns array of PostalZone objects by specified country.
+     *
+     * @param string $countryCode
+     * @param string $lang
+     *
+     * @return \Packlink\BusinessLogic\Http\DTO\PostalZone[]
+     *
+     * @throws \Logeecom\Infrastructure\Http\Exceptions\HttpAuthenticationException
+     * @throws \Logeecom\Infrastructure\Http\Exceptions\HttpCommunicationException
+     * @throws \Logeecom\Infrastructure\Http\Exceptions\HttpRequestException
+     */
+    public function getPostalZones($countryCode, $lang = 'en')
+    {
+        $url = 'locations/postalzones/destinations?' . http_build_query(
+                array(
+                    'platform' => 'PRO',
+                    'platform_country' => $countryCode,
+                    'language' => $lang,
+                )
+            );
+
+        $response = $this->call(HttpClient::HTTP_METHOD_GET, $url);
+
+        $postalZones = PostalZone::fromArrayBatch($response->decodeBodyAsJson());
+
+        $postalZones = array_filter(
+            $postalZones,
+            function ($postalZone) use ($countryCode) {
+                /** @var PostalZone $postalZone */
```

Comment:
Why is commented this code?

Reply:
What code is commented? This is a documentation comment.